### PR TITLE
release-notify: add --print option to print the location of the announcement file

### PR DESCRIPTION
**Pull request by cpanato**
Adds an option to print only the location of the Announcement file for publish manually

fixes: https://github.com/kubernetes/release/issues/958

EDIT:
command output for example using the 1.17 release
```bash
root@ubuntu-s-1vcpu-1gb-fra1-01:~/release# ./release-notify v1.17.0 --print
release-notify: BEGIN main on ubuntu-s-1vcpu-1gb-fra1-01 Wed Dec 11 18:04:12 UTC 2019

Skipping security_layer::auth_check...
Checking/setting cloud tools: OK
Setting global variables: OK
You can find the Announcement file here: gs://kubernetes-release-gcb/archive/anago-v1.17.0/announcement.html
To copy to your local machine use: /usr/bin/gsutil cp gs://kubernetes-release-gcb/archive/anago-v1.17.0/announcement.html .

release-notify: DONE main on ubuntu-s-1vcpu-1gb-fra1-01 Wed Dec 11 18:04:13 UTC 2019 in 1s
```

/sig release
/area release-eng
/kind feature
/priority important-soon
/milestone v1.18
/cc @kubernetes/release-engineering
/assign @justaugustus

### CODE REVIEW

File: lib/releaselib.sh [hunk 1]
```diff
@@ -1145,6 +1145,12 @@ release::send_announcement () {
   ((FLAGS_nomock)) || mailto=$GCP_USER
   mailto=${FLAGS_mailto:-$mailto}
 
+  if ((FLAGS_print)); then
+    echo "You can find the Announcement file here: $archive_root/announcement.html"
+    echo "To copy to your local machine use: $GSUTIL cp $archive_root/announcement.html ."
+    return 0
```
Comment (saschagrunert) [inline lib/releaselib.sh]: [I assume that this results in an `exit 0` and this is intended]

Comment (cpanato) [reply lib/releaselib.sh]: if you look the others in the same function it uses `return` using the same here as well

**Comment by cpanato** [comment]
@justaugustus will change :( overlook the issue, my bad

**Comment by justaugustus** [comment]
No problem, Carlos! :heart:

**Comment by cpanato** [comment]
@justaugustus applied the requested changes, let me know what do you think

**Comment by cpanato** [comment]
the announcement can be huge, so i dont know if is good to output the content of that in the screen or save in the clipboard (but that will need the `xclip` or other tool)

sample:

```
root@ubuntu-s-1vcpu-1gb-fra1-01:~/release# ./release-notify v1.17.0 --print
release-notify: BEGIN main on ubuntu-s-1vcpu-1gb-fra1-01 Fri Dec 13 13:39:02 UTC 2019

Skipping security_layer::auth_check...
Checking/setting cloud tools: OK
Setting global variables: OK
***** Start of Subject ******
k8s v1.17.0 is live!
***** End of Subject ******
***** Start of Announcement ******
Kubernetes team,
<P>
Kubernetes v1.17.0 has been built and pushed.
<P>
The release notes have been updated in <A HREF=https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.17.md/#v1170>CHANGELOG-1.17.md</A> with a pointer to it on <A HREF=https://github.com/kubernetes/kubernetes/releases/tag/v1.17.0>github</A>:
<P>
<HR>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type=text/css> table,th,tr,td {border: 1px solid gray; border-collapse: collapse;padding: 5px;} </style>
</head>
....
***** End of Announcement ******
***********
You can find the Subject file here: gs://kubernetes-release-gcb/archive/anago-v1.17.0/announcement-subject.txt
You can find the Announcement file here: gs://kubernetes-release-gcb/archive/anago-v1.17.0/announcement.html
***********
```

**Comment by cpanato** [comment]
done @hoegaarden thanks for the review

**Comment by justaugustus** [comment]
Thanks Carlos!
/lgtm
/approve
/hold cancel

**Pull request merged**